### PR TITLE
Add simpleblack theme

### PR DESCRIPTION
**Pull request by lucasprag**
Hi, great work on this plugin =)

I had this theme sitting in my own configuration for awhile and I was wondering if it would be nice to add to lightline's available themes.

This is how it looks like:
simpleblack

It was based on my colorscheme called [simpleblack](https://github.com/lucasprag/simpleblack)

I'm open to any feedback.

Thanks :)

### CODE REVIEW

**Comment by lucasprag** [comment]
I can't assign you as a reviewer, so I'm pinging you @itchyny =)

File: autoload/lightline/colorscheme/simpleblack.vim [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+let s:gray3 = [ '#4e4e4e', '239' ]
+let s:gray4 = [ '#626262', '241' ]
+
+let s:violet = [ '#CF73E6', '170' ]
+
+let s:blue = [ '#5f87af', '67' ]
+let s:blue2 = [ '#91AADF', '110' ]
+
+let s:green = [ '#57BA37', '71' ]
+let s:gold = [ '#F0D50C', '220' ]
```
Comment (itchyny) [inline autoload/lightline/colorscheme/simpleblack.vim]: Can you make the hex consistently lowercase?

Comment (lucasprag) [reply autoload/lightline/colorscheme/simpleblack.vim]: Sure, thanks for reviewing =)

Comment (lucasprag) [reply autoload/lightline/colorscheme/simpleblack.vim]: Sure, thanks for reviewing @itchyny =)

**Comment by itchyny** [comment]
Thank you.

**Comment by lucasprag** [comment]
Cool, thanks =)

**Pull request merged**